packages/api-sdk/package.json

```diff
@@ -44,7 +44,8 @@
     "lint": "npx eslint . --ext .ts",
     "lint-fix": "npm run lint --fix",
     "prettier-fix": "npx prettier '**/*.ts' --write",
-    "test": "echo \"No test specified yet\""
+    "test": "jest --runInBand --detectOpenHandles --passWithNoTests",
+    "test:e2e": "E2E=true jest --runInBand --detectOpenHandles"
```

Comment:
I don't think we'll need/implement E2E tests on the SDKs, or am I missing something?

Reply:
Good point, i copied this from api